web/styles/subscriptions.css

```diff
@@ -1425,12 +1420,11 @@ div.settings-radio-input-parent {
 }
 
 @media (width < $lg_min) {
-    .user-groups-container,
-    .subscriptions-container {
+    .two-pane-settings-container {
         max-width: 95%;
     }
 
-    .subscriptions-container .left .list-toggler-container {
+    .two-pane-settings-container .left .list-toggler-container {
```

Comment:
The usergroup container is less likely to wrap, but it seems good to give them shared styles.